packages/denokv/src/mod.ts

```diff
@@ -93,6 +94,35 @@ export class DenoKvStore implements KvStore {
       if (result.ok) return true;
     }
   }
+
+  /**
+   * {@inheritDoc KvStore.list}
+   * @since 1.10.0
+   */
+  async *list(prefix?: KvKey): AsyncIterable<KvStoreListEntry> {
+    const prefixToUse = prefix ?? [];
+
+    // First, check if the exact prefix key exists (only if prefix is specified)
+    if (prefix != null && prefix.length > 0) {
+      const exactEntry = await this.#kv.get(prefix);
+      if (exactEntry.value != null) {
```

Comment:
This excluding null value pattern isn't found in other integrations and is only visible here. Unless there's a specific issue with DenoKV, I think it would be appropriate to either modify `@fedify/fedify`'s `federation/kv.ts` to apply to all KV stores, or simply remove it here as well.